### ISSUE TITLE
Number of source documents strange

### ISSUE THREAD

**Issue by Alio241** [PromtEngineer/localGPT#234]
Hi, i was wondering is it normale that localGPT gives me only 4 source documents no matter what i ask him ? even tho i have more than 200 documents ?

**Comment by PromtEngineer** [comment]
Yes, the default is set to 4. Keep in mind, its 4 chunks, not actual documents.

**Comment by Alio241** [comment]
Do you know how i can change that ?
